Modules/Devices/test/Devices.Domain.Tests/Identities/StartDeletionProcessAsSupportTests.cs

```diff
@@ -50,6 +52,22 @@ public void Only_one_active_deletion_process_is_allowed_when_started()
         acting.Should().Throw<DomainException>().Which.Code.Should().Be("error.platform.validation.device.onlyOneActiveDeletionProcessAllowed");
     }
 
+    [Fact]
+    public void Starting_a_deletion_process_triggers_a_IdentityDeletionProcessStartedDomainEvent()
```

Comment:
We're already in `StartDeletionPrcessAsSupportTests`. So you can get rid of the first part of the test name:
```suggestion
    public void Triggers_an_IdentityDeletionProcessStartedDomainEvent()
```